daemon/daemon_unix.go

```diff
@@ -73,6 +73,7 @@ const (
 	// constant for cgroup drivers
 	cgroupFsDriver      = "cgroupfs"
 	cgroupSystemdDriver = "systemd"
+	cgroupNoneDriver    = "none"
```

Comment:
Could you update the swagger accordingly? https://github.com/moby/moby/blob/784bd2c883342c75a10c68d489d2de80fc50cb54/api/swagger.yaml#L3817-L3823

Also; 

- This might need a note to the API history https://github.com/moby/moby/blob/master/docs/api/version-history.md (but mention that it's not versioned)
- Does this need a mention in the "rootless" document? https://github.com/moby/moby/blob/master/docs/rootless.md 

Also wondering if this should be `none`, or if we should use an empty string for this (`""`)?

Reply:
updated.

> Also wondering if this should be none, or if we should use an empty string for this ("")?

I don't have a strong opinion, but I followed `--net=none` convention.

Reply:
Makes sense